src/consumption.cpp

```diff
@@ -366,6 +367,12 @@ edible_rating player::can_eat( const item &food, bool interactive, bool force )
         return ROTTEN;
     }
 
+    if( nutr > 0 && has_effect( effect_nausea ) ) {
```

Comment:
Description states drinking, but what you're checking is nutrition rather than eat/drink type. Which one is it?